sdk/identity/identity/test/node/deviceCodeCredential.spec.ts

```diff
@@ -346,11 +346,33 @@ describe("DeviceCodeCredential", function() {
               children: [
                 {
                   name: "Azure.Identity.DeviceCodeCredential-sendDeviceCodeRequest",
-                  children: []
+                  children: [
+                    {
+                      children: [],
+                      name: "core-http"
+                    }
+                  ]
                 },
                 {
                   name: "Azure.Identity.DeviceCodeCredential-pollForToken",
-                  children: []
+                  children: [
+                    {
+                      children: [],
+                      name: "core-http"
+                    },
+                    {
+                      children: [],
+                      name: "core-http"
+                    },
+                    {
+                      children: [],
+                      name: "core-http"
+                    },
+                    {
+                      children: [],
+                      name: "core-http"
+                    }
```

Comment:
Apologies for poking, but was wondering why it's repeated 4x here?

Reply:
That's because the test polls a fake endpoint for the device code auth response, so you see 4 requests go through core-http.  I'll add a comment!